l10n_it_fatturapa_in/models/company.py

```diff
@@ -12,38 +13,45 @@ class ResCompany(models.Model):
     sconto_maggiorazione_product_id = fields.Many2one(
         'product.product', 'Discount Supplement Product',
         help="Product used to model ScontoMaggiorazione XML element on bills."
-        )
+    )
+
+    arrotondamenti_attivi_account_id = fields.Many2one(
+        'account.account', 'Rounding Up Account',
+        domain=[('deprecated', '=', False)],
+        help="Account used to rounding up amount on bills."
+    )
+    arrotondamenti_passivi_account_id = fields.Many2one(
+        'account.account', 'Rounding Down Account',
+        domain=[('deprecated', '=', False)],
+        help="Account used to rounding down amount on bills."
```

Comment:
In questo caso dovrebbe essere usata la forma `Round up account`, `Round down account`. Vedi ad es.:

https://kb.sambapos.com/en/2-3-10-a-how-to-create-bill-rounding/

Stessa cosa direi per la forma verbale:
`...used to round up(down) amount on bills.`